.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+test.md
```

Comment:
- [ ] I see you ignored test.md, this is fine. You don't need to change anything here, just change the name of the file extension to `.md` not `.dm`